content/ADR-206-backpack-v2-back-end-dependencies.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: adr
+adr: 206
+title: Back-end dependencies for the new Backpack V2
+date: 2023-04-17
+status: Review
+type: RFC
+spdx-license: CC0-1.0
+authors:
+- sandrade-dcl
+- lorux0
+- davidejensen
+---
+
+## Need
+This is a technical proposal for the needs described here: [PRD: Backpack V2](https://www.notion.so/decentraland/PRD-Backpack-V2-13fd81cca29a45a180af6b73c2ca49b5)
+
+![image](https://user-images.githubusercontent.com/64659061/229762895-ea269990-2b40-412b-be26-071228f6a226.png)
+
+The intention of this document is:
+
+- Identifying which teams will be involved and what responsibilities they’ll cover.
+- Dividing the new functionality in smaller tasks that will be estimated accordingly in order to create a development roadmap.
+- Finding possible corner cases in order to preemptively identify and solve them.
+- Reaching a consensus between different affected teams in terms of defining the different functionalities and flows.
+
+### Involved teams
+This initiative will require a cross team effort that will include the following teams:
+
+- Explorer: for the UI, UI functionalities and integration with the back-end side.
+- Platform: for the modification of the already existing lambdas endpoints in order to make them compatible with the new requirements (sorting wearables by different criteria, filtering them by different criteria, etc.) and/or the creation of new ones required.
```

Comment:
Asking @menduz here. Should an ADR include information about which teams are going to develop the feature? This might be better suited for an internal document that can be used later to organize the roadmap and the development.

Reply:
I have adapted the document to the template and have removed this part 👍